index.html

```diff
@@ -216,6 +216,59 @@ <h1>Skills</h1>
                 </div>
             </div>
         </section>
+        <section class="contact-me">
+            <img id="contact-background" src="./images/mobile_geometry_1.svg" alt="background-box">
+            <div class="contact-me-form">
+                <h1> Get started</h1>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur
+                    adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna.
+                </p>
+                <form action="https://formspree.io/f/xwkyewre" method="post">
+                    <ul class="form-list">
+                        <li>
+                            <label for="name">Your name</label>
```

Comment:
- [ ] I think it would be nice to give the form a `class` and make it `display: flex`. This way, you would be practicing how to use the flexbox in laying out items as well as fulfilling one of the major project requirements. 👍